interop/applier.go

```diff
@@ -137,7 +137,7 @@ func addChainDependency(ctx context.Context, chain config.Chain, chainId uint64)
 		return fmt.Errorf("failed to read l1 block abi: %w", err)
 	}
 
-	cfgTypeAddDep := uint8(1)
+	cfgTypeAddDep := uint8(0)
```

Comment:
this enum changed: https://github.com/ethereum-optimism/optimism/pull/13921/files#diff-20fac0e056c186f0af3ee3aee13d444e1f451b8fb45eba4c4c56000549d32f0eL5